app/src/androidTest/kotlin/at/bitfire/davdroid/HiltTestRunner.kt

```diff
@@ -22,6 +26,12 @@ class HiltTestRunner : AndroidJUnitRunner() {
     override fun onCreate(arguments: Bundle?) {
         super.onCreate(arguments)
 
+        // set root logger to adb Logcat
```

Comment:
Will this show more detailed logs in CI as well? Would be really cool.

Reply:
Unfortunately only in adb logcat. But useful nevertheless to see what the BatchOperations etc. really do during the tests.